src/scala/com/github/johnynek/bazel_deps/Commands.scala

```diff
@@ -95,7 +108,12 @@ object Command {
       "check-only",
       help = "if set, the generated files are checked against the existing files but are not written; exits 0 if the files match").orFalse
 
-    (repoRoot |@| depsFile |@| shaFile |@| buildifier |@| checkOnly |@| Verbosity.opt).map(Generate(_, _, _, _, _, _))
+
+    val disable3rdPartyInRepos = Opts.flag(
```

Comment:
I tend to think this should be a function of the configuration file. What are your thoughts? Like, this is always false when you have external_repo set?

Reply:
Thats a good question -- i had an `OutputMode` option in the configuration file earlier, but it seemed like maybe you'd want to output both in a migration? and that having the target-file command line arg without the OutputMode set that way seemed strange. Mostly it seemed like i'd be cross validating 2 sets of options since the output path for the targetFile should be on the command line.

I'm open to bringing it back/doing it in there. Have really flip flopped about on this one

Reply:
okay, we can leave for now.